### PR TITLE
1 too short

### PR DESCRIPTION
**Pull request by 3djc**
Fix compilation issue since #5442

### CODE REVIEW

File: radio/src/stamp.cpp [hunk 1]
```diff
@@ -56,8 +56,8 @@ const char* getOtherVersion()
 }
 #elif defined(STM32)
 
-__attribute__ ((section(".fwversiondata"), used)) const char firmware_version[32] = "opentx-" FLAVOUR "-" VERSION " (" GIT_STR ")";
-__attribute__ ((section(".bootversiondata"), used)) const char boot_version[32] = "opentx-" FLAVOUR "-" VERSION " (" GIT_STR ")";
+__attribute__ ((section(".fwversiondata"), used)) const char firmware_version[33] = "opentx-" FLAVOUR "-" VERSION " (" GIT_STR ")";
```
Comment (bsongis) [inline radio/src/stamp.cpp]: why do you give a length here? the compiler should do it for you!

File: radio/src/stamp.cpp [hunk 2]
```diff
@@ -56,8 +56,8 @@ const char* getOtherVersion()
 }
 #elif defined(STM32)
 
-__attribute__ ((section(".fwversiondata"), used)) const char firmware_version[33] = "opentx-" FLAVOUR "-" VERSION " (" GIT_STR ")";
-__attribute__ ((section(".bootversiondata"), used)) const char boot_version[33] = "opentx-" FLAVOUR "-" VERSION " (" GIT_STR ")";
+__attribute__ ((section(".fwversiondata"), used)) const char firmware_version[] = "opentx-" FLAVOUR "-" VERSION " (" GIT_STR ")";
+__attribute__ ((section(".bootversiondata"), used)) const char boot_version[] = "opentx-" FLAVOUR "-" VERSION " (" GIT_STR ")";
```
Comment (raphaelcoeffic) [inline radio/src/stamp.cpp]: @schwabe didn't you assume some fixed size somewhere while reading that tag? I'm not sure anymore.

Comment (3djc) [reply radio/src/stamp.cpp]: Yeah, I was worried about that too, that's why I took the PR route

Comment (schwabe) [reply radio/src/stamp.cpp]: @raphaelcoeffic I would assume so too. But everything seems fine.

**Pull request merged**